packages/samples/headless-react/src/components/search-box/search-box.fn.tsx

```diff
@@ -0,0 +1,47 @@
+import {useEffect, useState, FunctionComponent} from 'react';
+import {
+  buildSearchBox,
+  SearchBox as HeadlessSearchBox,
+  SearchBoxOptions,
+} from '@coveo/headless';
+import {engine} from '../../engine';
+
+interface SearchBoxProps {
+  searchBox: HeadlessSearchBox;
```

Comment:
Suggestion:
To call this `controller` (for every component/sample) so that it gets a bit less confusing with the keyword `Searchbox` used everywhere.

For example, at the bottom of this file:
```
const options: SearchBoxOptions = {numberOfSuggestions: 8};
const searchBox = buildSearchBox(engine, {options});

<SearchBox searchBox={searchBox} />;
```
to this: 
```
const options: SearchBoxOptions = {numberOfSuggestions: 8};
const controller = buildSearchBox(engine, {options});

<SearchBox controller />;
```

Reply:
Hmm, I find phrases like `this.searchBox.submit()` easier to read compared to `this.controller.submit()`. However, I saw the same approach in Jean-Felix's decorator and Benjamin seems to agree, so maybe it is just me. I will make the change.